lite/examples/sound_classification/android_compose/app/build.gradle

```diff
@@ -0,0 +1,79 @@
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
+    id 'de.undercouch.download'
+}
+
+android {
+    compileSdk 31
+
+    defaultConfig {
+        applicationId "org.tensorflow.lite.examples.soundclassifier.compose"
+        minSdk 23
+        targetSdk 30
```

Comment:
Could you target SDK 31? It will save you time maintaining it after

Reply:
Thanks!